segmentstore/server/src/test/java/io/pravega/segmentstore/server/tables/TableCompactorTests.java

```diff
@@ -273,9 +273,6 @@ private void testCompaction(int readLength) {
         // In the real world, the IndexWriter will readjust this number as appropriate when reindexing these values.
         Assert.assertEquals("Expecting TOTAL_ENTRY_COUNT to be 0 after a full compaction.",
                 0, context.indexWriter.getTotalEntryCount(context.segmentMetadata));
-
-        Assert.assertEquals("Unexpected number of candidate reads from the segment with copy-on-read enabled.",
-                expectedCopyOnReadCount, context.segment.getCopyOnReadCount());
```

Comment:
probably you can drop `expectedCopyOnReadCount` variable

Reply:
Thanks @eolivelli, we couldn't address this comment as we received instructions to cut the 0.8 RC branch in the next hours (the PR author is not likely to be able to address that within this time frame). We will address your comment anyway in 0.9, if this works for you.

Reply:
@RaulGracia 
Good for me. it is not a big deal to leave or drop such useless variable.

It is also weird that spotbugs did not catch the dead local store.
I thought we were running spotbugs even against tests.